scala/bigtable-shc/src/main/scala/com/example/bigtable/spark/shc/AgeSource.scala

```diff
@@ -16,68 +16,79 @@
 
 package com.example.bigtable.spark.shc
 
-import org.apache.spark.sql.SparkSession
-import org.apache.spark.sql.execution.datasources.hbase.HBaseTableCatalog
-
-case class PersonRecord(
-                           name: String,
-                           age: Int)
-
 /**
   * This is a companion example with people name and ages, which may be
   * more concrete and easier to understand than `BigtableSource`.
   */
-object AgeSource {
+object AgeSource extends App {
+
+  val tableName = args(0)
+  val cat =
+    s"""{
+       |"table":{"namespace":"default", "name":"$tableName", "tableCoder":"PrimitiveType"},
+       |"rowkey":"name",
+       |"columns":{
+       |"name":{"cf":"rowkey", "col":"name", "type":"string"},
+       |"age":{"cf":"age", "col":"age", "type":"int"}
+       |}
+       |}""".stripMargin
+  val appName = getClass.getSimpleName.replace("$", "")
+
+  import org.apache.spark.sql.SparkSession
```

Comment:
We generally put all of the imports at the top.  Is it different for Spark?

Reply:
Not really Spark, but Scala in my hands :) Scala allows for imports to be closer to their use so it's easier for me to know what package a type belongs to. I have no strong opinion how it should be so if you feel sensitive about it, I'm fine to change it to your style.